recipes/mini-racer/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+schema_version: 1
+
+context:
+  name: mini-racer
+  version: 0.12.4
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/mini_racer-${{ version }}.tar.gz
+  sha256: 84c67553ce9f3736d4c617d8a3f882949d37a46cfb47fe11dab33dd6704e62a4
+
+build:
+  number: 0
+  skip: match(python, "<3.13")
+  #The skip is just to show the built can be successfull
```

Comment:
```suggestion
```
Since this is now working, you can remove the skip and let it run out of disk space and fail.  The build artifacts don't get generated until the feedstock is actually created, and at that point each major Python version is built in a different runner, so disk space won't be an issue.